requirements-dev-base.txt

```diff
@@ -16,6 +16,7 @@ feather-format>=0.4.1  # all_tests
 flake8==3.8.3  # lint
 flask>=1.0.0 # for s3 test only
 freezegun>=0.3.15  # all_tests
+gcsfs>=0.5.1 # all_tests
```

Comment:
The Google SDK has an implicit subdependency on `gcsfs` to actually read blob contents using the `gs://` URL format. I couldn't find it noted anywhere in the documentation but I ran into errors without it.

Let's keep tabs on this one as I don't have a lot of faith in these `fs` libs after that whole Azure fiasco